index.html

```diff
@@ -4080,11 +4080,9 @@ <h3>Parsing content</h3>
 <!-- - - - - - - - - - - - - Security and Privacy - - - - - - - - - - - - - -->
 <section> <h2 id="security">Security and Privacy</h2>
   <p>
-    NFC technology involves multiple levels of security. Payments done with NFC
-    are considered to be secure at hardware level, but the whole software stack
-    needs to be security hardened.
```

Comment:
I would add a separate sentence, like

"As general security measure,  the whole software stack needs to be security hardened".

And place where you think it adds most clarity/value.

Reply:
This seems obvious to me and not specific to Web NFC, don't you think?

Reply:
IIRC security reviewers insisted on stating the obvious :), but it's the editors' call, indeed.

Reply:
WDYT of https://github.com/w3c/web-nfc/pull/518/commits/ae9e5471a4ef736b1a1311ba19558fbd17e6685b